client/src/app/shared/components/sorting-tree/sorting-tree.component.ts

```diff
@@ -553,20 +671,20 @@ export class SortingTreeComponent<T extends Identifiable & Displayable> implemen
      * @param nextLevel The next level the node should have.
      * @param verticalMove The direction of the movement in the vertical way.
      */
-    private moveItemToTree(
-        node: FlatNode,
-        previousIndex: number,
-        nextIndex: number,
-        nextLevel: number,
-        verticalMove: Direction.UPWARDS | Direction.DOWNWARDS | Direction.NOWAY
-    ): void {
+    private moveItemToTree(node: FlatNode<T>, previousIndex: number, nextIndex: number, nextLevel: number): void {
+        let verticalMove =
+            previousIndex < nextIndex
+                ? Direction.DOWNWARDS
+                : previousIndex > nextIndex
+                ? Direction.UPWARDS
+                : Direction.NOWAY;
```

Comment:
The usage of one ternary operator for a statement is often okay. If you require two, just take `if-else`.